lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -473,6 +473,10 @@ def connect(dbapi_connection, connection_record):
 
     update(..., mysql_limit=10, mariadb_limit=10)
 
+* DELETE with LIMIT::
+
+    delete(..., mysql_limit=10, mariadb_limit=10)
```

Comment:
would need a `.. versionadded::` in docs

Reply:
this needs to be after the delete, not before